src/components/Offer.js

```diff
@@ -48,15 +48,16 @@ export const Offer = ({ data, navigation }) => {
   const link = sourceUrl?.url;
   const rootRouteName = navigation.getParam('rootRouteName', '');
   const headerTitle = navigation.getParam('title', '');
+  const dataProviderLogo = dataProvider?.logo?.url;
 
   // action to open source urls
   const openWebScreen = useCallback(
-    (webUrl) =>
+    () =>
```

Comment:
> * removed unnecessary arg from onPress function
>   * the function was only ever used as onPress, which does not get a string as an argument

does this also apply when passing down to other components? maybe it will be called with a param somewhere there?

```
{contentBlocks?.map((contentBlock, index) => {
  return (
    <StorySection
      contentBlock={contentBlock}
      index={index}
      key={`${contentBlock.id}-${index}`}
      openWebScreen={openWebScreen}
    />
  );
})}
{!!contact && (
  <Wrapper>
    <InfoCard contact={contact} openWebScreen={openWebScreen} />
  </Wrapper>
)}
<OperatingCompany
  title={texts.pointOfInterest.operatingCompany}
  logo={logo}
  operatingCompany={operatingCompany}
  openWebScreen={openWebScreen}
/>
```

Reply:
you are right, it is for example used in `InfoCard`.

I missed that it was provided as a prop to multiple children.

I reverted the change.

Good catch!